app/services/acuant/request.rb

```diff
@@ -56,15 +56,29 @@ def send_http_post_request
       end
     end
 
+    # rubocop:disable Metrics/MethodLength
     def faraday_connection
+      retry_options = {
+        max: 2,
+        interval: 0.05,
+        interval_randomness: 0.5,
+        backoff_factor: 2,
+        retry_statuses: [404, 438, 439],
```

Comment:
Nice, I didn't realize Faraday had retries built in :+1:

Reply:
Yeah. The documentation on it isn't great, but it does appear to work based on the tests, though we'll probably have to home in on the best way to use it and all that.